docs/embeddable-uis/embedding-elements.mdx

```diff
@@ -42,7 +42,12 @@ Additionally, you can manage the default settings of the element and customize t
 - Configure webhooks to receive automatic updates on actions taken by your users. For more information about webhooks, refer to
   the [Permit Webhooks documentation](/embeddable-uis/webhooks).
 
-_![iFrame visualization](/img/elements/user-management/IFrameDesign.png)_
+_![iFrame visualization](/img/elements/user-management/element-settings.png)_
+
+:::note Hide Add User Field
```

Comment:
As you moved it, you can just place it in a bullet above - after the webhooks section